carto/maps_api.py

```diff
@@ -0,0 +1,69 @@
+from .core import CartoException
+from secret import API_KEY, USER, EXISTING_TABLE, IMPORT_FILE, IMPORT_URL
+
+import requests
+
+
+IMPORT_API_FILE_URL = 'v1/map/named/'
+
+
+class NamedMap(object):
+
+	def __init__(self, client, template_name, params_name):
```

Comment:
indentation is wrong in the whole file, please make sure you use 4 spaces for indenting (PEP8)